src/jws/sign.rs

```diff
@@ -96,6 +96,14 @@ mod tests {
             PAYLOAD,
         );
 
+        _sign_works::<K256KeyPair>(
```

Comment:
I don't understand reason for this test. For jws payload as any string. So the test looks useless to me.